src/components/common/YourCompany/CorrectName/CorrectNameRequest.vue

```diff
@@ -244,13 +244,13 @@ export default class CorrectNameRequest extends Mixins(CommonMixin, NameRequestM
     const isConversionOrAlterationNameRequest = nr.request_action_cd === NrRequestActionCodes.CONVERSION
     let dialogContent = ''
     if (isConversionOrAlterationNameRequest) {
-      dialogContent = `<p class="info-text">This alteration name request from
-        ${GetCorpFullDescription(nr.entity_type_cd)} to ${GetCorpFullDescription(nr.legalType)}
-        does not match the current business type
+      dialogContent = `<p class="info-text">
+        This alteration name request does not match the current business type
         <b>${GetCorpFullDescription(this.getEntityType)}</b>.\n\n
-        The Name Request type must match the business type before you can continue.</p>`
+        The Name Request type must match the busines type before you can continue.</p>`
     } else {
-      dialogContent = `<p class="info-text">This ${GetCorpFullDescription(nr.entity_type_cd)}
+      dialogContent = `<p class="info-text">
+      This ${GetCorpFullDescription(nr.legalType)}
       Name Request does not match the current business type
       <b>${GetCorpFullDescription(this.getEntityType)}</b>.\n\n
       The Name Request type must match the business type before you can continue.</p>`
```

Comment:
Funny that lint didn't complain that lines 253-256 should be indented...? I think they should. (Let me know if lint says they shouldn't.)

Reply:
just fixed indention. and re-run Lint before fix, it doesn't complain any.